examples/graupnerbrunelstdp/stdputil.py

```diff
@@ -15,6 +15,22 @@
 
 logging.basicConfig(level=logging.WARN)
 
+#Note: having debug logging statements increases the run time by ~ 25%,
+# because they exist in tight loops, and expand their outputs, even when
+# debug is off, so we disable logging if possible.  Set this to true if
```

Comment:
but you're saying that they logging.debug executes code even when the debugging is turned off ?